cmd/cli/install.go

```diff
@@ -362,14 +362,14 @@ func (i *installCmd) validateOptions() error {
 
 	// Enforce single mesh cluster if needed
 	if i.enforceSingleMesh {
-		if len(list.Items) != 0 {
+		if len(osmControllerDeployments.Items) != 0 {
 			return errors.Errorf("Meshes already exist in cluster. Cannot enforce single mesh cluster.")
 		}
 	}
 
 	if i.deployPrometheus {
 		if !i.enablePrometheusScraping {
-			fmt.Fprintf(i.out, "Prometheus scraping is disabled. To enable it, set prometheus_scraping in %s/%s to true.\n", settings.Namespace(), constants.OSMConfigMap)
+			_, _ = fmt.Fprintf(i.out, "Prometheus scraping is disabled. To enable it, set prometheus_scraping in %s/%s to true.\n", settings.Namespace(), constants.OSMConfigMap)
```

Comment:
These we can safely ignore. We shouldn't have the linter complain about this and I think that's the case with `golangci-lint`.